src/client/components/index.ts

```diff
@@ -0,0 +1,15 @@
+// import LoginForm from './login_form';
+// import LoggedIn from './logged_in';
+import Home from './home';
+// import Home from './home';
+// import Home from '../../../js/client/components';
+// import Login from './login/login';
+// import Logout from './login/logout';
+
```

Comment:
glad this is working - we will comment this back in once we have the others in tsx format